site/content/en/docs/manual/basics/authorization.md

```diff
@@ -4,40 +4,70 @@ linkTitle: 'Authorization'
 weight: 1
 ---
 
-- First of all, you have to log in to CVAT tool. For authentication, you can use your username or email
-  you provided during registration.
+To start to annotate in CVAT, you need to create an account or log in to the existing account.
 
-  ![](/images/image001.jpg)
+To do this, use [App CVAT](https://app.cvat.ai/) page or install CVAT locally.
 
-- For register a new user press "Create an account"
 
-  ![](/images/image002.jpg)
 
-- You can register a user but by default it will not have rights even to view
-  list of tasks. Thus you should create a superuser. The superuser can use
-  [Django administration panel](http://localhost:8080/admin) to assign correct
-  groups to the user. Please use the command below to create an admin account:
+See:
 
-  ```bash
-    docker exec -it cvat_server bash -ic 'python3 ~/manage.py createsuperuser'
-  ```
+- [User registration and account access for non-admin users](#user-registration-and-account-access-for-non-admin-users)
+- [Register as a superuser](#register-as-a-superuser)
+
+
+## User registration and account access for non-admin users
+
+Open the login page:
 
-- If you want to create a non-admin account, you can do that using the link below
-  on the login page. Don't forget to modify permissions for the new user in the
-  administration panel. There are several groups (aka roles): admin, user,
-  annotator, observer.
+ ![](/images/image001.jpg)
+
+To register as a non-admin user, do the following:
+
+1. Click **Create an account**.
+
+   ![](/images/image002.jpg)
```

Comment:
What do you think about adding full image for login form?

Reply:
@klakhov good idea, updated